tests/test_plottingfuncs.py

```diff
@@ -24,6 +24,15 @@ def test_plot_and_save_colorbar(minicircle_pixels: Filters, plotting_config: dic
     )
     return fig
 
+@pytest.mark.mpl_image_compare(baseline_dir="resources/img/")
+def test_plot_and_save_colorbar_false(minicircle_pixels: Filters, plotting_config: dict, tmpdir) -> None:
```

Comment:
@ns-rse is this necessary due to a very similar, almost identical test beginning at line 10?